composer.json

```
@@ -15,15 +15,15 @@
         "forum": "https://discourse.laminas.dev/"
     },
     "require": {
-        "php": "^7.3 || ~8.0.0",
+        "php": "^7.3 || ~8.0.0 || ~8.1.0",
         "laminas/laminas-db": "^2.12",
         "laminas/laminas-paginator": "^2.10"
     },
     "require-dev": {
-        "laminas/laminas-coding-standard": "~2.1.4",
+        "laminas/laminas-coding-standard": "~2.3.0",
         "phpunit/phpunit": "^9.3",
```

```suggestion
        "phpunit/phpunit": "^9.5.8",
```
bumping phpunit would make it compatible in all combinations (7.4, 8.0, 8.1, lowest, locked, latest)